packages/flutter/lib/src/material/checkbox.dart

```diff
@@ -30,10 +30,10 @@ import 'toggleable.dart';
 ///  * <https://www.google.com/design/spec/components/selection-controls.html#selection-controls-checkbox>
 ///  * <https://www.google.com/design/spec/components/lists-controls.html#lists-controls-types-of-list-controls>
 class Checkbox extends StatelessWidget {
-  /// Constructs a checkbox
+  /// Creates a material design checkbox
   ///
-  /// * `value` determines whether the checkbox is checked.
-  /// * `onChanged` is called whenever the state of the checkbox should change.
+  /// * [value] determines whether the checkbox is checked.
+  /// * [onChanged] is called whenever the state of the checkbox should change.
```

Comment:
Whenever the -value- of the checkbox should change?

Developers, particularly the ones who've used non-React style toolkits, may be surprised to discover that it's up to them to maintain the checkbox's value.  Maybe we should say as much after "Creates a .. checkbox".